### PR TITLE
fix: Interactive config for plugins with no settings

### LINKED ISSUE #6890: bug: Can't configure freshdesk-tap
### Meltano Version

2.7.2

### Python Version

3.9

### Bug scope

CLI (options, error messages, logging, etc.)

### Operating System

Windows 10

### Description

Trying to set up freshdesk tap but getting error.
image (1)

### Code

_No response_

### PR DESCRIPTION
**Pull request by WillDaSilva**
Closes #6890

### CODE REVIEW

File: src/meltano/cli/interactive/config.py [hunk 1]
```diff
@@ -299,17 +299,24 @@ def configure(self, name, index=None, last_index=None, show_set_prompt=True):
 
     def configure_all(self):
         """Configure all settings."""
+        numeric_choices = [idx for idx, _, _ in self.setting_choices]
+        if not numeric_choices:
+            click.secho("There are no settings to configure.", fg="yellow")
```
Comment (aaronsteers) [inline src/meltano/cli/interactive/config.py:304]: @WillDaSilva - Let me see if there is any docs link we can provide here for guidance.

Comment (aaronsteers) [reply src/meltano/cli/interactive/config.py:304]: This is the link we provide on the Hub for defining/overriding settings:

- https://docs.meltano.com/guide/configuration#overriding-discoverable-plugin-properties

Here are two other related links but both lack context on exactly the steps to resolve:

- https://docs.meltano.com/guide/configuration#custom-settings
- https://docs.meltano.com/reference/plugin-definition-syntax#settings

Comment (aaronsteers) [reply src/meltano/cli/interactive/config.py:304]: Given that this is a pretty common issue and I don't feel great about any of these resources being very targeted or helpful, I'm leaning towards a new "No Settings Defined" section (or similar) and a shortcut URL: https://melta.no#no-settings-defined.

cc @tayloramurphy

**Comment by aaronsteers** [comment]
@WillDaSilva, @tayloramurphy, @sbalnojan - I added a link which will render at:

- https://melta.no#no-plugin-settings-defined

The content there can change over time but I wanted to get this included so we are directing users to specific set of resources for this problem.

File: docs/src/_guide/troubleshooting.md [hunk 2]
```diff
@@ -68,6 +68,20 @@ meltano            | DEBUG Invoking: ['demo-project/.meltano/loaders/target-json
...
+Related reference information and guides:
+
+- [Overriding discoverable plugin properties](https://docs.meltano.com/guide/configuration#overriding-discoverable-plugin-properties)
+- [Plugin `settings` syntax reference](https://docs.meltano.com/reference/plugin-definition-syntax#settings)
+- [Custom settings guide](https://docs.meltano.com/guide/configuration#custom-settings)
+
+After adding definitions for the plugin's settings, you may then configure those settings as usual with the `--interactive` option in `meltano config`.
+
+You may also contribute back to the community by publishing the settings to [MeltanoHub](https://hub.meltano.com) in the form of a new pull request.
+
```
Comment (WillDaSilva) [inline docs/src/_guide/troubleshooting.md:84]: @aaronsteers I won't approve this since it's my own PR, but this doc update looks good to me.

**Review by aaronsteers — APPROVED** [review]

File: docs/src/_guide/troubleshooting.md (hunk 2 above)
Comment (aaronsteers) [reply docs/src/_guide/troubleshooting.md:84]: Thanks, @WillDaSilva ! @tayloramurphy is out at Coalesce so let's go ahead and merge. 🚀

**Pull request merged**